statements/src/main/java/org/lukawska/trainsmart/statements/domain/valueObjects/AgreementStatus.java

```diff
@@ -0,0 +1,7 @@
+package org.lukawska.trainsmart.statements.domain.valueObjects;
+
+public enum AgreementStatus {
```

Comment:
ja bym to skrocil do jednej linijki. Pod warunkiem ze defaultowo intellij Ci tego tak nie formatuje.